camel-cdi-integration/src/main/java/io/silverware/microservices/providers/camel/CamelCdiContextFactory.java

```diff
@@ -62,8 +61,7 @@ public CamelContext createCamelContext(final Context context) {
          if (cdiSilverService.isDeployed()) {
             log.info("CDI SilverService connected successfully!");
 
-            CamelContextMap camelContextMap = cdiSilverService.findByType(CamelContextMap.class);
-            return camelContextMap.getCamelContext("camel-1");
+            return cdiSilverService.findByType(CamelContext.class);
```

Comment:
I have just run our quickstarts with these changes and it seems that Camel uses `camel-1` context name by default. So the behavior has not been changed here.